File: Backend/App/models/obstetrical_history.py
from sqlalchemy import Column,String,Integer,Boolean,ForeignKey
from Backend.App.core.database import Base
from Backend.App.models.case_history import CaseHistory

class ObstetricalHistory(Base):
    __tablename__ ="Obstetrical_History"

    ObstetricalHistoryId = Column(Integer,primary_key=True,index=True)
    CaseId=Column(Integer,ForeignKey(CaseHistory.CaseId))
    History = Column(String(255))

File: DotNetBackend/ViveksHomoeoClinic/ViveksHomoeoClinic/Models/EFModel/ObstetricalHistory.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ViveksHomoeoClinic.Models.EFModel
{
    public class ObstetricalHistory
    {
        [Key]
        public int ObstetricalHistoryId { get; set; }
        [Required]
        public int CaseId { get; set; }

        [ForeignKey(nameof(CaseId))]
        public CaseHistory Case { get; set; }
        [Required]
        public string Description { get; set; }
    }
}
